src/test/groovy/net/vivin/gradle/versioning/TagTaskSpecification.groovy

```diff
@@ -166,27 +166,43 @@ class TagTaskSpecification extends Specification {
         annotated << [false, true]
     }
 
-    @Unroll
-    def 'tag is skipped if #tagAndPushName is executed (annotated: #annotated)'() {
+    @Unroll("#testName")
+    def 'created annotated tag has expected message'() {
         given:
-        testRepository.origin = origin
-
-        and:
+        System.getenv()
         testRepository
             .makeChanges()
-            .commitAndTag('0.0.1', annotated)
+            .commitAndTag("1.0.0", true)
```

Comment:
Since I was testing for annotated tags with messages, I decided to use this. Is that what you meant?